src/LightXML.jl

```diff
@@ -2,24 +2,23 @@ __precompile__()
 
 module LightXML
 
-using Compat; import Compat.String
-
-# iteratorsize is new in 0.5, declare it here for older versions. However,
-# we do not actually support calling these, since the traits are not defined
-if VERSION < v"0.5.0-dev+3305"
-    function iteratorsize(v)
-        error("Do not call this on older versions")
-    end
+using Compat
+
+@static if VERSION < v"0.7.0-DEV"
+    const IteratorSize = Base.iteratorsize
 else
-    import Base: iteratorsize, SizeUnknown, IsInfinite, HasLength
+    import Base: IteratorSize
```

Comment:
this isn't defined for early 0.7-dev, also does not need at-static

Reply:
OK, I wasn't able to get `@compat` to fix that for me, not sure what the right magic incantation is :grin: